Components/grid/grid.css

```diff
@@ -0,0 +1,125 @@
+@import url("../../main.css");
+@import url('../../colors.css');
+@import url('../../typography.css');
+
+
+/* colors for unit borders */
+:root {
+    --color1: #001219;
+    --color2: #005f73;
+    --color3: #3d9397;
+    --color4: #94d28d;
+    --color5: #e9d8a6;
+    --color6: #ee9800;
+    --color7: #ca6702;
+    --color8: #bb3e03;
+    --color9: #ae2012;
+    --color10: #9b2226;
+    --color11: rgba(255, 255, 255, 0.8);
+}
+
+/* general web-page styles */
+#collections {
+    margin: 0 auto;
+    padding: 0 var(--space-md);
+}
+
+#collections ul{
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+}
+
+#collections h2 {
+    font-size: calc(2.5 * var(--paragraph));;
+}
+
+
+/* styling the figure containers */
+#collections figure{
+    max-height: 400px;
+    overflow: hidden;
+    position: relative;
+    border-width: 10px;
+    border-style: solid;
+    border-radius: calc(0.05 * var(--width-unit));
+    margin: var(--space-xs) 0;
+}
+
+#collections figure img{
+    max-width: 100%;
+    margin-bottom: -4px;
+    object-fit: cover;
+}
+
+#collections figcaption{
+    position: absolute;
+    bottom: 0;
+    background-color: var(--color11);
+    width: 100%;
+    opacity: 0.7;
+    font-size: var(--header1);
+}
+
+#collections figcaption a {
+    display: block;
+    padding: var(--space-xs);
+    text-decoration: none;
+    color: var(--color1);
+}
+
+/* declaring border for figure and hover for figcaption */
+.color2, .color2 a:hover {
+    border-color: var(--color2);
+    color: var(--color2) !important;
```

Comment:
you are using ! important, it is a bad practice you can remove it.

Reply:
yes I know but the border style is overriding so I have to use it to maintain consistency